cmd/brew-brewdle.rb

```diff
@@ -1,5 +1,39 @@
 #!/usr/bin/env ruby
 
 BREWDLER_ROOT = File.expand_path "#{File.dirname(__FILE__)}/.."
-ENV["RUBYLIB"] = "#{BREWDLER_ROOT}/lib:#{ENV["RUBYLIB"]}"
-exec "#{BREWDLER_ROOT}/bin/brewdle"
+BREWDLER_LIB = Pathname.new(BREWDLER_ROOT)/"lib"
+
+$LOAD_PATH.unshift(BREWDLER_LIB)
+
+require "brewdler"
+
+usage = <<-EOS.undent
+  brew brewdle
+  brew brewdle [--version]
+  brew brewdle [-h|--help]
+
+  Usage:
+  Bundler for non-Ruby dependencies from Homebrew
+
+  Options:
+  -h, --help        show this help message and exit
+  --version         show the version of brewdler
+EOS
+
+if ARGV.include?("--version")
+  puts Brewdler::VERSION
+  exit 0
+end
+
+if ARGV.flag?("--help")
```

Comment:
I think this is inevitable. Since we drop the gem, the homebrew env will definite be there. Say we implement ARGV extend here again, the homebrew extension will still exist. In the result, it would be hard to see which code is acutely running, homebrew or brewdler. 

Also for the heads up, [my implement](https://github.com/xu-cheng/homebrew-brewdler/compare/Homebrew:master...xu-cheng:bundle) of #31 requires homebrew internal method too. Namely, I need the `version`.